src/neptune/new/exceptions.py

```diff
@@ -1334,3 +1335,14 @@ def __init__(self):
 {correct}Need help?{end}-> https://docs.neptune.ai/getting_help
 """
         super().__init__(message.format(**STYLES))
+
+
+class StreamAlreadyUsedException(NeptuneException):
+    def __init__(self):
+        message = """
+{h1}
+----StreamAlreadyUsedException---------------------------------------------------
+{end}
+TODO: add docs
```

Comment:
@normandy7 we could have some information explaining user that he can log file created with `File.from_stream` at most once.
It includes case when user invoked `file.content` or `file._save()` before logging it.